### DISCUSSION TITLE
Http error 431 (header size is to large) when redirect user to Account Console.

### DISCUSSION THREAD

**Discussion by husamnadeesh** [keycloak/keycloak#29162] [Q&A] [answered]
Hey there. I'm using Keycloak as my IDP for .Net Core MVC app, users can access Account Console to change profile info and credentials  when the mvc app uses http protocol but in case of https protocol they got http error 431( header size is to large) knowing that my headers are only ~15K.

**Comment by codespearhead** [comment]
Are you running Keycloak in development mode (`bin/kc.sh start-dev`) or production mode (`bin/kc.sh start`)?

Are you running Keycloak behind a reverse proxy? If so you have to [configure it accordingly](https://github.com/keycloak/keycloak/discussions/28279#discussioncomment-9001818).

**Reply by husamnadeesh** [reply]
I'm running Keycloak in development mode without any reverse proxy.

**Reply by codespearhead** [reply, marked as answer]
The current workaround is to pass enviroment variable `QUARKUS_HTTP_HTTP2=false` to the container, but this issue is being investigated in #10236 .

If that doesn't work, tell us what Keycloak version you're running and how and where you're deploying Keycloak.

Possible duplicates: #10236 , #13933 .

**Reply by husamnadeesh** [reply]
Thanks  a lot it worked properly.
